### PR TITLE
Initial multimodal support

### PR DESCRIPTION
**Pull request by ricklamers**

### CODE REVIEW

File: packages/openassistants-react/tsconfig.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "lib": ["dom", "esnext"],
     "importHelpers": true,
     // output .d.ts declaration files for consumers
-    "declaration": true,
+    "declaration": false,
```
Comment (ricklamers) [inline packages/openassistants-react/tsconfig.json:9]: I'm getting: 

(typescript) Error: .../openassistants/packages/openassistants-react/src/components/ui/button.tsx(7,7): semantic error TS2742: The inferred type of 'buttonVariants' cannot be named without a reference to 'class-variance-authority/node_modules/clsx'. This is likely not portable. A type annotation is necessary.

Comment (ricklamers) [reply packages/openassistants-react/tsconfig.json:9]: @colemccracken did you get this before? Any ideas how to fix? Do we need declartion outputs?

Comment (ricklamers) [reply packages/openassistants-react/tsconfig.json]: github.com/joe-bell/cva/issues/163

Ah, they don't export types. I don't particularly like the solution I've committed (copying their types). Maybe we should look for an alternative.

Comment (colemccracken) [reply packages/openassistants-react/tsconfig.json]: Seems fine. Sounded like he intended to fix in 1.0 which is currently in beta so we can probably remove on upgrade

**Review by colemccracken — APPROVED** [review]
Great work!

**Pull request merged**